### PR TITLE
hide csv fields if user wont upload to youtube

### PR DESCRIPTION
**Pull request by rycastaneda**

### CODE REVIEW

**Comment by ravenjohn** [comment]
<a href="http://ninja.raven.tm/anyTV/music-dashboard/pull/393" target="_blank">ReviewNinja

File: app/scripts/config.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ module.exports = {
         components: './components/'
     },
     url: {
-        api: 'http://api.tunes.tm',
```
Comment (jvmsangkal) [inline app/scripts/config.js]: don't include this !fix

**Comment by jvmsangkal** [comment]
:AllenApproves:

File: app/scripts/config.js (hunk 1 above)
Comment (jvmsangkal) [inline app/scripts/config.js]: !fixed

**Comment by sibayanjasper** [comment]
:AllenApproves:

**Pull request merged**